pom.xml

```diff
@@ -3,16 +3,23 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.580</version>
+    <version>2.2</version>
   </parent>
 
+  <properties>
+    <jenkins.version>1.577</jenkins.version>
```

Comment:
It is generally recommended to use a Jenkins LTS baseline or release as the version you depend on. Compared to 1.580, there's just ~300 instances left that report usage statistics that would be affected by this.